operations/services.md

```diff
@@ -1579,6 +1579,311 @@ Returns all companionships based on customers, reservations or reservation group
+            "SerialNumber": null,
+            "ValidityStartUtc": "2020-10-09T22:00:00Z",
+            "ValidityEndUtc": "2020-10-10T22:00:00Z",
+            "Permissions":
+            {
+                "Bed": { "Value": true },
+                "Room": { "Value": false },
+                "Floor": { "Value": false },
+                "Building": { "Value": false }
+            }
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ResourceAccessTokenParameters` | Array of [Resource access token parameter](#resource-access-token-parameter) | required, max 1000 items | Parameters of [Resource access token](#resource-access-token). |
+
+#### Resource access token parameter
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ServiceOrderId` | string | required | Unique identifier of a service order (for example [Reservation](reservations.md#reservation)). |
+| `CompanionshipId` | string | optional | Unique identifier of [Companionship](services.md#companionship). |
+| `ResourceId` | string | optional | Unique identifier of [Resource](enterprises.md#resource). |
+| `Type` | [Resource access token type](#resource-access-token-type) | required | Type of stored value. |
+| `Value` | string | required | Value of resource access token |
+| `SerialNumber` | string | optional | Serial number of [Resource access token type](#resource-access-token-type). |
+| `ValidityStartUtc` | string | required | Marks the start of interval in which the resource access token can be used. |
+| `ValidityEndUtc` | string | required | Marks the end of interval in which the resource access token can be used. |
+| `Permissions` | [Resource access token permission parameter](#resource-access-token-permission-parameter) | required | Specify permissions of [Resource access token](#resource-access-token). |
+
+#### Resource access token permission parameter
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Bed` | [Bool update value](reservations.md#bool-update-value) | optional | Specify whether [Resource access token](#resource-access-token) grants permission to access bed. |
+| `Room` | [Bool update value](reservations.md#bool-update-value) | optional | Specify whether [Resource access token](#resource-access-token) grants permission to access room. |
+| `Floor` | [Bool update value](reservations.md#bool-update-value) | optional | Specify whether [Resource access token](#resource-access-token) grants permission to access floor. |
+| `Building` | [Bool update value](reservations.md#bool-update-value) | optional | Specify whether [Resource access token](#resource-access-token) grants permission to access building. |
```

Comment:
Again, the specify pattern is weird, make it please consistent with rest of the documentation.